newIDE/app/src/EventsFunctionsExtensionsLoader/MetadataDeclarationHelpers.js

```diff
@@ -906,6 +916,30 @@ const declarePropertyInstructionAndExpression = (
       .addParameter('yesorno', i18n._(t`New value to set`), '', false)
       .getCodeExtraInformation()
       .setFunctionName(setterName);
+
+    addObjectAndBehaviorParameters(
+      entityMetadata.addScopedAction(
+        toggleActionName,
+        i18n._(t`Toggle ${propertyLabel}`),
+        i18n._(
+          t`Toggle the property value for ${'"' + uncapitalizedLabel + '"'}.`
```

Comment:
Toggle property Xxx. 

Is shorter and better I think.

Reply:
Do you mean showing the technical name in the sentence?